src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackSectionBlock.php

```diff
@@ -44,6 +44,11 @@ public function field(string $text, bool $markdown = true): self
             'text' => $text,
         ];
 
+        // Maximum number of items is 10
+        if (10 <= \count($this->options['fields'])) {
```

Comment:
should be `<`, with `<=` you throw not only for `count > 10` but also for `count == 10`

Reply:
Good catch, fixed in 7bac792328

Reply:
@guilliamxavier you're right! I have tested more over [here](https://app.slack.com/block-kit-builder/T5JPWDX2T#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22text%22:%22A%20message%20*with%20some%20bold%20text*%20and%20_some%20italicized%20text_.%22,%22type%22:%22mrkdwn%22%7D,%22fields%22:%5B%7B%22type%22:%22mrkdwn%22,%22text%22:%221%22%7D,%7B%22type%22:%22mrkdwn%22,%22text%22:%222%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%223%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%224%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%225%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%226%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%227%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%228%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%229%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%2210%22%7D,%7B%22type%22:%22plain_text%22,%22text%22:%2211%22%7D%5D%7D%5D%7D) and you're right!